src/main/java/com/pikachu/purple/bootstrap/rating/vo/RatingInfo.java

```diff
@@ -0,0 +1,14 @@
+package com.pikachu.purple.bootstrap.rating.vo;
+
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+
+public record RatingInfo(
+    Long perfumeId,
+    String perfumeName,
+    @Min(value = 1, message = "최소 값은 1입니다.")
+    @Max(value = 5, message = "최대 값은 5입니다.")
+    int score
```

Comment:
p3) 이건 @Range로 지정하면 될 것 같아요!
```suggestion
    @Range(min=1, max=5)
    int score
```